.eslintrc

```diff
@@ -18,7 +18,8 @@
     "filenames/match-exported": 0,
     "import/no-extraneous-dependencies": 0,
     "no-undef": 0,
-    "no-use-before-define": 0
+    "no-use-before-define": 0,
+    "no-shadow": 0

```

Comment:
Hello @UladzislauKutarkin  
How i understood we should use same eslint rules across all modules and modules should not have specific rules.
@artem-blazhko @zburke Could you please correct me if i am wrong?

Reply:
Hello @Dmitriy-Litvinenko , it's not my changes. It comes from another PR since my branch based on UIBULKED-305